src/components/store/store.tsx

```diff
@@ -12,10 +12,12 @@ import {
   createIngredientsSlice,
   createRecipesInCartSlice,
   createRecipesSlice,
+  createTemporaryUISlice,
   MyIngredientsState,
   MyRecipesInCartState,
   MyRecipesState,
-} from "./my-state";
+  MyTemporaryUIState,
```

Comment:
Dobrej naming, ted me napadlo, ze tenhle kus stavu mozna nebudeme chtit "ukladat" do toho persistent storage. https://github.com/pmndrs/zustand/blob/main/docs/integrations/persisting-store-data.md#partialize ale asi se tim ted nemusime trapit, jen me to napadlo diky tvemu dobremu namingu :)